api-admin/src/main/java/bio/terra/pearl/api/admin/controller/forms/ConsentFormController.java

```diff
@@ -39,4 +39,16 @@ public ResponseEntity<Object> newVersion(String portalShortcode, String stableId
         consentFormExtService.createNewVersion(portalShortcode, consentForm, adminUser);
     return ResponseEntity.ok(savedConsent);
   }
+
+  @Override
+  public ResponseEntity<Object> create(String portalShortcode, String stableId, Object body) {
+    AdminUser adminUser = authUtilService.requireAdminUser(request);
+    ConsentForm consentForm = objectMapper.convertValue(body, ConsentForm.class);
+    if (!stableId.equals(consentForm.getStableId())) {
```

Comment:
we would never expect (unless there's a bug) from someone using our UI.  It would only happen if someone is accessing the API directly, in which case they'd fix it by correcting the invalid parameter